airflow/providers/apache/hive/CHANGELOG.rst

```diff
@@ -27,6 +27,14 @@
 Changelog
 ---------
 
+7.0.2
```

Comment:
I think it's a breaking change it should be 8.0.0

Reply:
Sure, made the change for that